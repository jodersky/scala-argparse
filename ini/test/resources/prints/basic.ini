[section1]
foo=bar
a.b=c
