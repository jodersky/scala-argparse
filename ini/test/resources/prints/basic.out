
[section1]
  foo=bar

[section1.a]
  b=c
